src/main/scala/se/nimsa/dcm4che/streams/DicomParsing.scala

```diff
@@ -214,6 +215,9 @@ trait DicomParsing {
   def intToBytesBE(i: Int): ByteString = ByteString((i >> 24).toByte, (i >> 16).toByte, (i >> 8).toByte, i.toByte)
   def intToBytesLE(i: Int): ByteString = ByteString(i.toByte, (i >> 8).toByte, (i >> 16).toByte, (i >> 24).toByte)
 
+
```

Comment:
Will it not be more unified and Scala-esque to stay in line with the methods above (`intToBytes`)? That is, drop the use of `ByteUtils` and implement our own, ByteString-based method with signature `shortToBytes(s: Short, bigEndian: Boolean)'. The `offset` and `bytes` parameters are unnecessary when using immutable ByteStrings.